libvips/module/meson.build

```diff
@@ -0,0 +1,16 @@
+module_sources = files(
+    'jxl.c',
+    'heif.c',
+    'magick.c',
+    'openslide.c',
+    'poppler.c',
+)
+
+libvips_sources += module_sources
+
+module_lib = static_library('module',
```

Comment:
This probably needs to be `shared_module` instead of `static_library`. It also needs a few other changes (such as depending on `libvips_dep`, etc.) and including the `name_suffix: module_suffix` workaround:
```meson
module_suffix = []
# Keep the autotools convention for shared module suffix because GModule
# depends on it: https://gitlab.gnome.org/GNOME/glib/issues/1413
if ['darwin', 'ios'].contains(host_machine.system())
  module_suffix = 'so'
endif
```

Reply:
I've now implemented the module handling